paper-server/src/main/java/org/bukkit/craftbukkit/entity/CraftArmadillo.java

```diff
@@ -14,8 +16,47 @@ public net.minecraft.world.entity.animal.armadillo.Armadillo getHandle() {
         return (net.minecraft.world.entity.animal.armadillo.Armadillo) super.getHandle();
     }
 
+    @Override
+    public State getState() {
+        return this.stateToBukkit(this.getHandle().getState());
```

Comment:
stateToBukkit and stateToNMS are static methods so it's incorrect to have "this" here